### ISSUE TITLE
Enhancement: RSI period setting

### ISSUE THREAD

**Issue by Utone** [GuntharDeNiro/BTCT#255]
UPDATE:

TLDR: adjusting sensitivity to mach tame frames is probably not reliable as i see it cross over on 1 minute not on 5 and again do crossover on 15 minutes on some pairs.. BUTTT...

Profits guys 9 day period seems to be the shit, read here:

https://www.dailyfx.com/forex/education/trading_tips/trend_of_the_day/2014/08/27/3-Trading-Tips-for-RSI.html
no idea what this and MFI can do together but we like to do shit no one else has done before am i right bXT gunbotters????

I assume we currently use the common standard 14 day period RSI?

After noticing RSI level changes with different time periods and tweaking with scale in tradingview, because I assumed the same ultimate low or high peak would result in the same RSI value across time periods.  Nope of course not..: lol That would make it too easy right??

In general the longer the timeframe the less RSI is affected:
https://s.tradingview.com/x/y5LrsNqX/ see how that first small dip would trigger a bad buy signal?
https://s.tradingview.com/x/O3Yua94H/
https://s.tradingview.com/x/Jg2JKpI1/
https://s.tradingview.com/x/RVT14MNl/ at 30 minutes you might get lucky and get a buy
https://s.tradingview.com/x/f5XIvcs2/ Here at 1 hour it isn't oversold..

But I did find something interesting: RSI period setting.
You could use this to tune the sensitivity, so that even if you change your Time Period used from, say 15 to 5 minutes you can check in advance what time period levels out the difference in sensitivity so it best corresponds to the 5 minute output.

And educing the RSI period as low as 2 days results in way more buy and sell signals Basically turning it into Stoch RSI so for us short term traders using 15, 5 or even 1 minute periods a lower RSI period (or at least the ability to adjust the sensitivity would greatly improve the indicators profitability..
And use it as a more dependable indicator and we can tune it to a coins behaviour (in override) so it matches up the general strategy (used as a catch all pairs, handy for when market changes) 

Very often this setting can greatly improve a coins performance with RSI based strategies.
I have not dived into accuracy settings but i suspect it to be subtle but they are available in the tradingview default RSI indicator. (now also used in Sjakil's Zeno GUI)

http://tradingmarkets.com/recent/why_rsi_may_be_one_of_the_best_short-term_indicators_-673367.html

**Comment by GunbotKitty** [comment]
Closing due to inactivity. Open a new issue if necessary.
